docs/src/pages/components/data-grid/sorting/sorting.md

```diff
@@ -4,34 +4,91 @@ title: Data Grid - Sorting
 
 # Data Grid - Sorting
 
-<p class="description">Sorting allows ordering records in the data grid.</p>
+<p class="description">Easily sort your rows based on one or several criteria.</p>
 
-## Basic sorting
+The sorting rules can be modified through the grid interface by clicking on a column header. Repeat this action to pass to the next sorting direction of the column.
 
-Single column sorting can be triggered with by clicking a column header.
-Repeat this action to change the sorting direction.
+{{"demo": "pages/components/data-grid/sorting/BasicExampleDataGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
-A sorted column can be can pre-configured using the `sortModel` prop:
+## Single and multi-sorting
```

Comment:
This section content is only about multi-sorting
```suggestion
## Single and multi-sorting [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/)
```

Reply:
Thanks. I think the preview is looking great.